module2/intermission_work/index.md

```diff
@@ -1,5 +1,61 @@
 ---
+title: Mod 2 Intermission Work
 layout: page
-title: Module 2 - Intermission Pre-Work
 ---
 
+## Learning Goals
+
+* Understand basic HTML elements and attributes
+* Understand basic CSS styling
+* Understand what a relational database is
+* Interact with a relational database by writing SQL
+* Explore the basics of Web Development
+  * Understand and Implement CRUD functionality
+  * Understand and implement the MVC design pattern
+* Begin learning the conventions of a Rails app
+
+## Deliverables:
+
+First, create a new [Gist on Github](https://gist.github.com/). Copy and paste the markdown from [this template](https://gist.github.com/mikedao/cf0f63490a0ef91ac7d251e95fc410f7) into your gist.
+
+This Gist will be your submission for your intermission work. Each of the assignments includes a Check for Understanding that you must complete in your Gist. Some of the assignments will also ask you to include a link to a GitHub repo.
+
+**Deliverables are due 5:00 pm the Saturday before the module begins. Prework must be completed in its entirety in order to be successful in Mod 2.**
+
+## Assignments
+
+Work on the following assignments in order. Each assignment will assume that you have completed the previous one and the accompanying Checks for Understanding. **Ruby 3.2.2 and Rails 7.1.2 are required.**
+
+Time-box recommendations are in each section's instructions, and also included here for convenience. 
+
+1. Sign up for the [GitHub Student Developer Pack](https://education.github.com/pack), using your `@turing.edu` email address. 
+   - This step sometimes takes a few days for GitHub to complete, but once you sign up for your credits you'll be able to take advantage of their benefits! 
+   - Follow [these instructions](https://mod4.turing.edu/lessons/intermission/gh-student-instructions.html) to get the info you require to complete registration. **You do NOT need to complete the Heroku steps at this time!** Ask in #codehelp if you get stuck. 
+2. [Get Module 2 Tools](./tools)
+3. [Intro to LinkedIn](./linkedin)
+4. Styling (2 hrs total)
+    * [Intro to HTML](./html) (1h)
+    * [Intro to CSS](./css) (1h)
+    * STRICTLY TIME BOX THESE EXPLORATIONS. You will only need to know extremely basic HTML and CSS in this mod.
```

Comment:
Let's remove this because now we are doing more HTML and CSS in this mod

Reply:
agreed -- removing and increasing the time estimations for this section (reducing for SQL)